src/pages/nerd-days.js

```diff
@@ -233,13 +232,16 @@ const NerdDaysPage = () => {
                   have any questions about Nerd Days please emails{' '}
                   <a href="mailto:deco@newrelic.com">deco@newrelic.com</a>.
                 </p>
-                <h3
-                  css={css`
-                    margin: 25px 0 0 0;
-                  `}
-                >
-                  Additional Nerd Days Events
-                </h3>
+              </div>
+              {/* <MarketoForm
+                id={4783}
+                title="REGISTER FOR NERD DAYS | AMERICAS"
+                munchkinId="412-MZS-894"
+                publishableKey="pk_4d10daa544de6f993a9a9ce002ccd1c6"
+                redirectLink="/nerd-days-thank-you"
+              /> */}
```

Comment:
```suggestion
```

Do we need any of this or would it be ok to just delete it?

Reply:
We can definitely delete this I think. It seems to be commented code which was removed from the form deletion

Reply:
@jaesius do you need this to stay?

Reply:
I'm going to go ahead and commit/merge this. We can always refer back tot his PR if we need to grab some data from this code block.